go/tools/builders/compilepkg.go

```diff
@@ -343,6 +344,17 @@ func compileArchive(
 			cgoSrcs[i-len(goSrcs)] = coverSrc
 		}
 	}
+	if strings.Contains(outLinkObj, "external/") && slices.Contains(gcFlags, "-d=libfuzzer") {
+		// Remove -d=libfuzzer from gcFlags when compiling external packages. We don't really want to instrument them,
+		// and they may not link without libfuzzer_shim.go.
+		gcFlags = slices.DeleteFunc(gcFlags, func(s string) bool {
+			return s == "-d=libfuzzer"
+		})
+	}
+	// Log instrumented objs for ease of tracking/debugging.
+	if slices.Contains(gcFlags, "-d=libfuzzer") {
+		fmt.Printf("%s -- gcFlags=%s\n", outLinkObj, gcFlags)
```

Comment:
I don't think we want this checked in.

Reply:
Yep, I'll remove; couldn't find a standard way of logging debug messages, which would get suppressed by default.